doc/howtos/building-images.rst

```diff
@@ -256,6 +284,51 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_FEATURES_append = "${OSTRO_IMAGE_FEATURES_DEV}"
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
 
+Adding a Custom Layer in Ostro OS
+---------------------------------
+
+.. _`Creating Your Own Layer`: http://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#creating-your-own-layer
+.. _`Open Embedded Layers Index`:  http://layers.openembedded.org/layerindex/branch/master/layers/
+
+The Yocto Project documentation explains the steps you'd follow for `Creating Your Own Layer`_. 
+
+The `Open Embedded Layers Index`_ is a database that's searchable by layer and recipe name.  For example
+if you wanted to add `opencv` (open computer vision layer) you can find the recipe there and also a list 
+of other layers it depends on.
+
+Only specific recipes from the layers in meta-openembedded are
+supported in combination with Ostro OS, even though all of
+meta-openembedded gets imported into the combined repository.
+
+To use recipes from meta-openembedded, they must be added to the
+respective ``PNWHITELIST`` variables, in ``conf/ostro/ostro.conf`` for officially supported ones
+or in a ``local.conf`` for unofficial ones in a personal build.  
+You can refer to ``conf/ostro/ostro.conf`` for more information about white- and black-listing.
+
+#. Within your
+   cloned copy of ``ostro-os``, here's how you can easily add a custom layer into your Ostro OS build:::
+
+      $ git clone <meta-custom-layer-name>     # clone the git repo for your custom layer 
+      $ source oe-init-build-env               # initialize the build environment 
+
+#. Add a line to your ``build/conf/bblayers.conf`` file adding the layer:::
```

Comment:
I asked Henry Bruce about this and he recommended a manual edit like this. Let me know if you'd really like something else here.

I’m willing to revisit my position on this but would like to understand why he is making such recommendation? I only see downsides to manually editing the file versus letting the tool handle all this.